virt.if

```diff
@@ -1759,3 +1777,22 @@ interface(`virt_svirt_manage_tmp',`
 	manage_dirs_pattern($1, svirt_tmp_t, svirt_tmp_t)
 	manage_sock_files_pattern($1, svirt_tmp_t, svirt_tmp_t)
 ')
+
+########################################
+## <summary>
+##     Read qemu PID files.
+## </summary>
+## <param name="domain">
+##     <summary>
+##     Domain allowed access.
+##     </summary>
+## </param>
+#
+interface(`qemu_read_pid_files',`
```

Comment:
Let's use "virt" prefix for interface name to make clear in which interface file is the interface declared.